cli/run-oz-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+echo PWD=$(pwd)
+
+function cleanup {
+  cd "$PWD"
+}
+trap cleanup
+
+cd ../openzeppelin-contracts || exit
+
+PROXY_URL="$1"
+NN="$2"
+DELAY="$3"
+TEST="$4"
+echo PROXY_URL="$PROXY_URL"
+echo SOLANA_URL="$SOLANA_URL"
+echo NN="$NN"
+echo DELAY="$DELAY"
+echo TEST="$TEST"
+sleep "$DELAY"
+date
+DD=$NN""$(date | tr -d '[:alpha:]: ')
+echo DD="$DD"
+curl -w '\n' -X POST -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","method":"web3_clientVersion","params":[],"id":'$DD'}' $PROXY_URL
```

Comment:
Usage of functions like [here](https://github.com/neonlabsorg/neon-e2e/blob/eb68a6239ea5b7437d3bcf30cc34a2dba2b2a52c/.github/actions/allure-environment-data/action.yml#L14) is also welcomed

Reply:
we will change if necessary in the future